libs/sdk-ui-dashboard/src/model/store/storeAccessors/DashboardStoreAccessorRepository.ts

```diff
@@ -0,0 +1,140 @@
+// (C) 2022 GoodData Corporation
+
+import { DashboardDispatch, DashboardSelectorEvaluator, DashboardState } from "../types";
+import { invariant } from "ts-invariant";
+import { DashboardStoreAccessor } from "./DashboardStoreAccessor";
+import { idRef, ObjRef, serializeObjRef } from "@gooddata/sdk-model";
+
+/**
+ * This singleton class serves the selector and the dispatcher properties for given dashboard {@link @gooddata/sdk-model#IdentifierRef}.
+ *
+ * The {@link Dashboard} component has an optional property {@link IDashboardProps#onStateChange} through which
+ * you can handle setting of the values for {@link DashboardDispatch} and {@link DashboardSelectorEvaluator}.
+ *
+ *
+ * In your component using {@link Dashboard} you can create an onStateChange callback for your dashboard using
+ * {@link DashboardStoreAccessorRepository#getOnChangeHandlerForDashboard} and pass it to mentioned Dashboard
+ * component property.
+ *
+ * @example
+ * ```
+ *  const dashboardStoreAccessor = DashboardStoreAccessorRepository.getInstance();
```

Comment:
The variable should probably be named with plural: it contains all the accessors, not just one.